### ISSUE TITLE
Cannot get Virtual Instance details

### ISSUE THREAD

**Issue by parente** [softlayer/softlayer-python#568]
Tried on 4.0.3.

```
slcli vs detail 5259406

SoftLayerAPIError(SoftLayer_Exception_WebService_ObjectMask): Property 'userData' not valid for 'SoftLayer_Virtual_Guest'.
```

Also tried back on 3.1.0 which used to work and its failing now too.

```
sl vs detail 5259406

SoftLayerAPIError(SoftLayer_Exception_WebService_ObjectMask): Property 'userData' not valid for 'SoftLayer_Virtual_Guest'.
```

Not sure if this is a breaking server-side API change that the client needs to adjust for or a snafu.

**Comment by parente** [comment]
MR fixes the issue, but still seems weird that this suddenly stopped working in the API.

**Comment by vincentwoo** [comment]
just started getting this problem, would appreciate a quick merge and fix

**Comment by c4po** [comment]
getting this problem as well.

**Comment by rmchamberlain** [comment]
Just started getting this problem - would love to have it fixed, as all of my SoftLayer administration tools are currently not operational.

**Comment by sudorandom** [comment]
A tentative fix for this has been released as a tag and on pypi as version 4.0.4.
